tests/test_browser_model.py

```diff
@@ -36,7 +38,16 @@ def test_close_window(foxpuppet):
 def test_switch_to(foxpuppet, selenium):
     """Test Switch to function"""
     foxpuppet.browser.open_window()
-
     # Switch to originally window opened by pytest
     foxpuppet.browser.switch_to()
     assert foxpuppet.browser.handle == selenium.current_window_handle
+
+
+@pytest.mark.firefox_preferences({'privacy.trackingprotection.enabled': True})
+def test_tracking_protection_shield(foxpuppet, selenium):
+    """Tests if the tracking protection icon displays"""
+    browser = foxpuppet.browser
+    assert not browser.navbar.is_tracking_shield_displayed
+    selenium.get('https://www.washingtonpost.com/')
+    assert WebDriverWait(selenium, timeout=5).until(
+        lambda _: browser.navbar.is_tracking_shield_displayed)
```

Comment:
Are you seeing intermittent failures when the tracking icon is not displayed? Generally I'd prefer not to have waits in the tests, but I think this makes the most sense at least for now. Note that you don't need the `assert` as the wait will either result in `True` or will raise a Timeout exception.